test/mlir/krnl/get_induction_var_value.mlir

```diff
@@ -0,0 +1,53 @@
+// RUN: onnx-mlir-opt --convert-krnl-to-affine %s -split-input-file | FileCheck %s
+
+func @test_simple() {
+  %ii = krnl.define_loops 1
+  krnl.iterate(%ii) with (%ii -> %i = 0 to 10) {
+    %idx = krnl.get_induction_var_value(%ii) : (!krnl.loop) -> (index)
```

Comment:
This is a loop, right, as `%ii` is already fully specified?

Reply:
Yes, this is a degenerate case where you can just use `i`. Instead of going to great length trying to convert the loop handler into an induction variable.